### DISCUSSION TITLE
Shareaza clone Envy...

### DISCUSSION THREAD

**Discussion by Mika3578** [ansani/Shareaza#60] [General]
Someone already use this? some interesting feature? can be implemented here, dark skin etc...
https://sourceforge.net/projects/getenvy/
https://github.com/GetEnvy/Envy

**Comment by ansani** [comment]
Hi @Mika3578 !
I checked sources and basically is a fork/clone stopped some years ago. I didn't see any interesting implementation (at least at protocol/feature level). Of course if you think something should be added, feel free to open an issue.

**Reply by Mika3578** [reply]
is not the most recently updated fork of shareaza?
